lib/plug/cowboy.ex

```diff
@@ -55,6 +56,34 @@ defmodule Plug.Cowboy do
   When using a unix socket, OTP 21+ is required for `Plug.Static` and
   `Plug.Conn.send_file/3` to behave correctly.
 
+  ## Loopback vs Public IP Addresses
+
+  Should your application bind to a loopback address, such as `::1` (IPv6) or
+  `127.0.0.1` (IPv4), or a public one, such as `::0` (IPv6) or `0.0.0.0`
+  (IPv4)?
+  It depends on how (and whether) you want it to be reachable from other
```

Comment:
Is this part of the same paragraph or a new one? If the latter, you need an extra newline. :)